Code/Framework/AzToolsFramework/AzToolsFramework/Editor/EditorContextMenuBus.h

```diff
@@ -11,6 +11,7 @@
 #include <AzCore/Outcome/Outcome.h>
 #include <AzCore/std/any.h>
 #include <AzCore/std/string/string.h>
+#include <AzCore/Math/Vector2.h>
```

Comment:
Would it not be faster to do
```
namespace AZ
{
    class Vector2;
}
```

So that hte header does not need to be included?

Reply:
Resolved